sdks/java/maven-archetypes/examples/src/test/resources/projects/basic/archetype.properties

```diff
@@ -15,7 +15,7 @@
 #    limitations under the License.
 #
 package=it.pkg
-version=0.1-SNAPSHOT
+version=0.1
```

Comment:
why this change? I think SNAPSHOT should be there.

But maybe it should be 0.2-SNAPSHOT?